cpp/src/barretenberg/honk/sumcheck/sumcheck.hpp

```diff
@@ -54,38 +56,34 @@ template <typename Flavor, class Transcript, template <class> class... Relations
                   \-|  *  |  *  |  *  |  *  | ... |  *  | vertex n-1
             m = n/2
                                         *
-            Each group consists of N edges |, and our construction of univariates and folding
+            Each group consists of N edges |, and our construction of univariates and partial evaluation
                                         *
             operations naturally operate on these groups of edges
 
     *
     * NOTE: With ~40 columns, prob only want to allocate 256 EdgeGroup's at once to keep stack under 1MB?
     * TODO(#224)(Cody): might want to just do C-style multidimensional array? for guaranteed adjacency?
     */
-    FoldedPolynomials folded_polynomials;
+    PartiallyEvaluatedMultivariates partially_evaluated_polynomials;
 
     // prover instantiates sumcheck with circuit size and a prover transcript
     Sumcheck(size_t multivariate_n, ProverTranscript<FF>& transcript)
         : transcript(transcript)
         , multivariate_n(multivariate_n)
         , multivariate_d(numeric::get_msb(multivariate_n))
-        , round(multivariate_n, std::tuple(Relations<FF>()...))
```

Comment:
This now happens in the constructor of `PartiallyEvaluatedMultivariates` :heavy_check_mark: